newsletter-june-2021.rst

```diff
@@ -0,0 +1,87 @@
+.. post:: June 3, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features from June.
```

Comment:
```suggestion
      our current focus, and upcoming features.
```

Lets us not touch this each month? :)

Reply:
I'm not sure, I feel bad about keeping the description always the same.